app/views/accounts/edit.html.slim

```diff
@@ -0,0 +1,21 @@
+h1 Edit user details
+p 
+  |User id: 
+  = @accounts.try(:id)
```

Comment:
If you decide to use helper  locals where you assign properly `:account`  then in the view you use `account` instead of `@account`